hack/tools/go.mod

```diff
@@ -33,14 +34,16 @@ require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/dennwc/varint v1.0.0 // indirect
 	github.com/digitalocean/godo v1.65.0 // indirect
+	github.com/dnephin/pflag v1.0.7 // indirect
```

Comment:
weird, normally it is https://github.com/spf13/pflag

Reply:
huh https://github.com/gotestyourself/gotestsum/commit/badd93420947c25800d558d3effbf9c5e05c888d

Reply:
Weird, but since the usage is restricted to gotestsum, it should be ok?